rent-book-api/app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+module Api::V1
+  class CommentsController < ApplicationController
```

Comment:
```suggestion
  class Api::V1::CommentsController < ApplicationController
```
k cần dùng module nữa

Reply:
dạ để em đổi